server/src/main/java/org/apache/cassandra/sidecar/cdc/CdcConfig.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.cassandra.sidecar.cdc;
+
+import java.util.Map;
+import org.apache.cassandra.sidecar.common.server.utils.MillisecondBoundConfiguration;
+import org.apache.cassandra.sidecar.common.server.utils.MinuteBoundConfiguration;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * In memory representation of CDC and Kafka configurations from "configs" table in sidecar keyspace.
+ */
+public interface CdcConfig
+{
+    /**
+     * @return returns environment of the cassandra cluster. This config could be optional
+     */
+    String env();
+
+    /**
+     * @return returns kafka topic for the mutations to be published
+     */
+    @Nullable
+    String kafkaTopic();
+
+    /**
+     * @return returns topic formats
+     */
+    @NotNull
+    TopicFormatType topicFormat();
+
+    /**
+     * @return returns if CDC is enabled or not
+     */
+    boolean cdcEnabled();
+
+    /**
+     * @return returns unique global identifier for CDC job, CDC state is associated with job-id
+     */
+    String jobId();
+
+    /**
+     * @return returns configurations of the kafka for the mutations to be published.
+     */
+    Map<String, Object> kafkaConfigs();
+
+    /**
+     * @return returns CDC configurations as a map
+     */
+    Map<String, Object> cdcConfigs();
+
+    /**
+     * @return if logOnly config is set, mutations will not be published to kafka, instead they
+     * would be logged, this would be useful for debugging and running sidecar application locally
+     */
+    boolean logOnly();
+
+    /**
+     * @return returns the data center, this config could be optional
+     */
+    String dc();
```

Comment:
NIT: for consistency with the codebase
```suggestion
    String datacenter();
```